ee/clickhouse/views/actions.py

```diff
@@ -83,19 +83,6 @@ def people(self, request: Request, *args: Any, **kwargs: Any) -> Response:  # ty
             }
         )
 
-    @action(methods=["GET"], detail=True)
-    def count(self, request: Request, **kwargs) -> Response:  # type: ignore
-        action = self.get_object()
-        query, params = format_action_filter(action)
-        if query == "":
-            return Response({"count": 0})
-
-        results = sync_execute(
-            "SELECT count(1) FROM events WHERE team_id = %(team_id)s AND {}".format(query),
```

Comment:
I think the action count is still quite helpful. How about doing something like count in the last 30 days?

Reply:
Use 3 months for count + table